### PR TITLE
Refining casing for Data Studios

### PR DESCRIPTION
**Pull request by justinegeffen**
Refining and improving casing for Data Studios.

### CODE REVIEW

File: platform_versioned_docs/version-24.1/data/data-studios.mdx [hunk 1]
```diff
@@ -89,61 +89,61 @@ The default user is the `root` account. To install extensions, select **Extensio
...
-1. From the list of data studios in your workspace, select the three dots next to the status message for the data studio you want to start, then select **Start as new**. The **Add data studio** form opens.
+1. From the list of sessions in your workspace, select the three dots next to the status message for the session you want to start, then select **Start as new**. The **Add data studio** form opens.
 1. Optional: In the **Add data studio** form, customize the following fields:
     - **Data studio name**
     - **Description**: By default, the parent data studio name(s).
     - **Mounted data**
     - **CPU**
     - **Memory**
-1. Select **Add** to add the new data studio.
+1. Select **Add** to add the new Data Studio.
```
Comment (robnewman) [inline platform_versioned_docs/version-24.1/data/data-studios.mdx:126]: This is a regression - it's not the Product. It should be "new session"

```suggestion
1. Select **Add** to add the new session.
```

**Review by robnewman — CHANGES_REQUESTED** [review]
Just one minor fix needed, otherwise looks good.

**Comment by jason-seqera** [comment]
I've manually applied these changes to: https://github.com/seqeralabs/docs/pull/258

**Review by robnewman — APPROVED** [review]

**Comment by jason-seqera** [comment]
We can therefore technically close this since the other PR replaces this.

**Pull request closed without merging**